app/templates/search/_categories.html

```diff
@@ -1,14 +1,47 @@
-<ul>
-  {% for category in categories %}
-    <li>
-      {% if category.link %}<a href="{{ category.link }}">{% endif %}{% if category.selected %}<strong>{% endif %}
-        {{ category.label }}
-      {% if category.selected %}</strong>{% endif %}{% if category.link %}</a>{% endif %}
-      {% if category.children and category.selected %}
-        {% with categories=category.children %}
-          {% include 'search/_categories.html' %}
-        {% endwith %}
+{%- macro top_level_link(text, url) -%}
+  <a href="{{ url }}" class="lot-filters__top-level-link"><span class="visually-hidden">Back to </span>{{ text }}</a>
```

Comment:
The user didn't necessarily get to the current page by going clicking through all of the levels, so I don't quite like 'back to'. Right now it's obviously the most likely way they got there, but I think in future we will want to have clever ways of encouraging people to search across specific categories (e.g. a suggested search based on keywords they typed). So if we do have to have some extra content for screen readers etc (I'm not sure I agree that this is necessary for accessibility, though), then some other text? "Up a level to" might actually explain the structure better.